scripts/stubsabot.py

```diff
@@ -3,40 +3,47 @@
 
 import argparse
 import asyncio
+import calendar
 import contextlib
 import datetime
 import enum
 import functools
 import io
 import os
 import re
+import shutil
 import subprocess
 import sys
 import tarfile
 import textwrap
 import urllib.parse
 import zipfile
-from collections.abc import Iterator, Mapping, Sequence
+from collections.abc import Callable, Iterator, Mapping, Sequence
 from dataclasses import dataclass, field
 from http import HTTPStatus
 from pathlib import Path
-from typing import Annotated, Any, ClassVar, NamedTuple
+from typing import Annotated, Any, ClassVar, NamedTuple, TypeVar
 from typing_extensions import Self, TypeAlias
 
 import aiohttp
 import packaging.version
+import tomli
 import tomlkit
 from packaging.specifiers import Specifier
 from termcolor import colored
+from tomlkit.items import String
 
-from ts_utils.metadata import StubMetadata, read_metadata, update_metadata
+from ts_utils.metadata import NoSuchStubError, StubMetadata, metadata_path, read_metadata, update_metadata
 from ts_utils.paths import STUBS_PATH, distribution_path
 
 TYPESHED_OWNER = "python"
 TYPESHED_API_URL = f"https://api.github.com/repos/{TYPESHED_OWNER}/typeshed"
 
 STUBSABOT_LABEL = "bot: stubsabot"
 
+PYRIGHT_CONFIG = Path("pyrightconfig.stricter.json")
```

Comment:
This should be moved to `ts_utils.paths` and be made relative to `TS_BASE_PATH`.